TradeItIosTicketSDK2/TradeItSelectBrokerViewController.swift

```diff
@@ -219,4 +222,18 @@ class TradeItSelectBrokerViewController: CloseableViewController, UITableViewDel
 
         return cell
     }
+
+    private func getRemoteLogo(forBroker broker: TradeItBroker, imageView: UIImageView) -> Bool {
```

Comment:
Well... maybe. Let me think about it. We don't really want people to have to implement/overwrite this logic as far as I know.